src/teleop/gui/src/components/AutonWaypointEditor.vue

```diff
@@ -203,9 +212,11 @@ export default {
         }
       }
 
-      const courseMsg = new ROSLIB.Message(course)
+      const course_request = new ROSLIB.ServiceRequest({
+        enableMsg: course
+      });
       
-      this.course_pub.publish(courseMsg)
+      this.course_pub.callService(course_request, (res) => {})
```

Comment:
Doesn't this need to set waitingForNav?

Reply:
This runs in an interval, waitingForNav is set elsewhere

Reply:
Oh I see, it's just generally set when we receive nav status. Okay in that case having an empty callback here makes sense